webapp/components/wpt-flags.js

```diff
@@ -227,11 +226,6 @@ class WPTFlagsEditor extends FlagsEditorClass(/*environmentFlags*/ false) {
         Use web-platform-tests.live.
       </paper-checkbox>
     </paper-item>
-    <paper-item>
-      <paper-checkbox checked="{{showMetadataInfo}}">
-        Show metadata Information on wpt.fyi result page.
-      </paper-checkbox>
-    </paper-item>
```

Comment:
What's going on here?

Reply:
This is a flag to indicate that the metadata information will be requested along with TestRuns, but we don't need it anymore